app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -78,11 +86,11 @@ export const ContentForm: React.FC<IContentFormProps> = ({
   const { isShowing: showNLPModal, toggle: toggleNLP } = useModal();
   const [{ sources, series, sourceOptions, productOptions }, { getSeries }] = useLookupOptions();
   const { combined, formType } = useCombinedView(initContentType);
-  const hub = useApiHub();
+  useScrollTo(id, scrollToContent ? 'bottom-pane' : '');
```

Comment:
@jdtoombs This is the new hook for scrolling based on the work you originally did.